pkg/porter/publish.go

```diff
@@ -25,8 +25,7 @@ import (
 type PublishOptions struct {
 	BundlePullOptions
 	bundleFileOptions
-	InsecureRegistry bool
-	ArchiveFile      string
+	ArchiveFile string
```

Comment:
I wasn't really expecting to see this type of change made in this PR? 

I realize now that InsecureRegistry is a shadow definition of BundlePullOptions.InsecureRegristry but it would be helpful to do this in a separate PR as it's not related to the scope of the current PR.

Reply:
Oh yup, this wasn't intended to be included!  Will remove.